From ecc45d979c7f44014b43dce91e943f7d6e32bfbd Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Tue, 23 Jun 2020 18:20:35 -0500
Subject: [PATCH] Remove System.Security.Principal.Windows nupkg ref

This package is available in the sharedfx and isn't necessary as a package
reference. The package ref blocks source build because it uses version 4.7.0,
built in the 3.1 release and not available for 2.1 source-build.

This fix is in PR in the upstream repo: https://github.com/dotnet/cli/pull/13408
---
 src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj b/src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj
index 575160565..6cade3da0 100644
--- a/src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj
+++ b/src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj
@@ -26,7 +26,6 @@
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCorePackageVersion)" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
     <PackageReference Include="XliffTasks" Version="$(XliffTasksPackageVersion)" PrivateAssets="All" />
-    <PackageReference Include="System.Security.Principal.Windows" Version="4.7.0" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETStandard' ">
-- 
2.26.2.windows.1

